.github/workflows/build-and-test-pr.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - uses: actions/setup-java@v3
         with:
           distribution: 'zulu'
-          java-version: 8
+          java-version: 17
```

Comment:
We want to ensure compatibility with JDK8, here should not update the jdk version, if necessary, should add a ci action

Reply:
Yeah, the reason for this is that it is the most convenient way to verify the support for Spring 6 and Spring Boot 3, and now it has been reverted.